### DISCUSSION TITLE
hiding relationship manager during Editing of the Parent record

### DISCUSSION THREAD

**Discussion by MarJose123** [filamentphp/filament#6760] [Help] [answered]
Is there a way to hide relationship manager when editing the parent record?

I saw this documentation, but only applicable on record value not user action.
https://filamentphp.com/docs/2.x/admin/resources/relation-managers#conditional-visibility

**Comment by danharrin** [comment, marked as answer]
Override `getRelationManagers()` on the page and return an empty array

**Reply by valetto** [reply]
how did u access parent record in getRelationManagers

**Reply by danharrin** [reply]
`$this->record`
